ethminer/MinerAux.h

```diff
@@ -542,6 +584,7 @@ class MinerCLI
 			if (m_minerType == MinerType::CUDA || m_minerType == MinerType::Mixed)
 				CUDAMiner::listDevices();
 #endif
+			stop_io_service();
```

Comment:
Can you try return from function instead of `exit()`.

Reply:
I did not change the logic of functions not related to the scope of this pr.
Exit was there before: I only addedd the stop_io_service before exiting.